proto/storage/discovered_cluster.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "storage/cluster.proto";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+// DiscoveredCluster is the storage representation of discovered clusters.
+//
+// Refer to v1.DiscoveredCluster for a more detailed doc.
+message DiscoveredCluster {
+  // UUIDv5 generated deterministically from the tuple (metadata.id, metadata.type, source.id).
```

Comment:
nit: For storage representation, you can drop the verbose commenting, but it's not required.